src/components/send/components/AccountIDInput.js

```diff
@@ -0,0 +1,212 @@
+import React, { Component, createRef } from 'react'
```

Comment:
<3 I love that this component doesn't have branching to handle corner cases around whether we're trying to create a new account with the input; the assumption that this is used for only accounts that already exist is really nice from that perspective.